File: DynamoDBQueries/put_item.py
from http import client
import boto3
client = boto3.client("dynamodb")

resp = client.put_item(
  TableName='MovieRoles',
  Item={
    'Actor': { 'S': 'Tom Hanks'},
    'Movie': { 'S': 'Toy Story'},
    'Role': { 'S': 'Woody'},
    'Year': { 'N': '1995'},
    'Genre': { 'S': "Children's"} 
  }
)

File: DynamoDBQueries/put_item.js
const AWS = require('aws-sdk')
const dynamodb = new AWS.DynamoDB()

const resp = dynamodb.putItem({
  TableName: 'MovieRoles',
  Item: {
    'Actor': { 'S': 'Tom Hanks' },
    'Movie': { 'S': 'Toy Story' },
    'Role': { 'S': 'Woody' },
    'Year': { 'N': '1995' },
    'Genre': { 'S': "Children's" }
  }
})
